src/text-style.ts

```diff
@@ -0,0 +1,23 @@
+import type { Schema } from "prosemirror-model";
+import mapValues from "just-map-values";
```

Comment:
We don't need to use this lib. This is so thin and makes the code even harder to interpret.